spec/controllers/teams_controller_spec.rb

```diff
@@ -40,7 +40,7 @@
 
       get :index
 
-      expect(response).to redirect_to logout_login_path
+      expect(response).to redirect_to sessions_path
```

Comment:
Shouldn't this redirect to the logout path? Or rather, add the action _destroy_?